src/models/BEPRO/Network.js

```diff
@@ -285,12 +278,16 @@ class Network extends IContract {
 
   /**
    * Change amount needed for Council
-   * @param {number} value
+   * @param {Object} params
+   * @param {number} params.value
    * @return {Promise<TransactionObject>}
    */
-  async changeCouncilAmount(value) {
-    return await this.__sendTx(
-      this.getWeb3Contract().methods.changeCOUNCIL_AMOUNT(Numbers.toSmartContractDecimals(value, this.getSettlerTokenContract().getDecimals())),
+  changeCouncilAmount({ value }, options) {
```

Comment:
web-network is not ready for this change, to change this argument type you'd need to do the same on a PR to web-network:master